shared/actions/config/index.js

```diff
@@ -390,6 +391,23 @@ const updateServerConfig = (state: TypedState) =>
     }
   })
 
+const writeLastSentXLM = (state: TypedState, action: SetLastSentXLMPayload) => {
```

Comment:
can use the shorthand ```=> action.payload.writeFile...```
should also catch, we don't care if this fails so lets not black bar